doc/ref_model/chapters/appendix-a.md

```diff
@@ -30,7 +30,7 @@ As Part of the [Transition Plan](../../gov/chapters/chapter09.md#9.2) described
 
 | Ref        	| Type       	| Name               	| Description                                                           	|
 |------------	|------------	|--------------------	|-----------------------------------------------------------------------	|
-| rm.exc.001 	| Technology 	| SRIOV 	| This exception allows workloads (VNF/CNF) to rely on SR-IOV technology. 	|
+| rm.exc.001 	| Technology 	| SRIOV 	| This exception allows workloads (VNF/CNF) to rely on SR-IOV over PCI-PT technology. 	|
```

Comment:
I would rather use PCI-Pass Through than PCI-PT, but not very strong opinion.

Reply:
@CsatariGergely Thanks Gergely. Updated. Pls approve. Thx, -Mark